ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -283,6 +282,12 @@ VirtualizationRootResult VirtualizationRoot_RegisterProviderForPath(PrjFSProvide
         vnode_put(virtualizationRootVNode);
     }
     
+    if (rootIndex >= 0)
```

Comment:
Should we reset the auth cache TTL back to the default upon virtualization root unmount?

Reply:
Yes we will certainly want to do that, but right now unmount doesn't call through to the kext at all so we're currently not doing any cleanup. I filed #284 to cover all of that.